src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -482,6 +484,17 @@ def inventory(entry: SettingsEntry, config: ApplicationConfiguration) -> PostPro
                 else:
                     # file path
                     entry.value.current.append(abs_user_path(inv_entry))
+
+        # Something may be required
+        if config.app == "inventory" and entry.value.current is C.NOT_SET:
+            if config.entry("help_inventory").value.current is False:
```

Comment:
Can the two `if` conditions be combined `if config.app == "inventory" and entry.value.current is C.NOT_SET and config.entry("help_inventory").value.current is False`

Reply:
yes, it looked a little wierd, so I'll set vars for the conditions